games-strategy/7kaa/7kaa-2.15.4p1.recipe

```diff
@@ -76,7 +79,10 @@ INSTALL() {
 	mkdir -p $appsDir
 	mkdir -p $dataDir
 	
-	mv data/ $dataDir/7kaa
+	
+	mkdir -p $postInstallDir
+	cp createHaikuConfig.sh $portDir/7kaa_postinstall.sh
+	install -t $postInstallDir -m 755 $portDir/7kaa_postinstall.sh
```

Comment:
the usual way of doing this is with:
`USER_SETTINGS_FILES="
        settings/kyua directory
        settings/kyua/kyua.conf template $dataDir/kyua/examples/kyua.conf"`

Reply:
ok, I'll replace that later

Reply:
the template is not automatically installed on pacakage installation. So I just re-added the post install script for now. Though this would probably clash if there would ever be multiuser support. Do you know how to do this without a script?

Reply:
Yeah sorry for that. USER_SETTINGS_FILES is a declaration with no automatic actions. GLOBAL_WRITABLE_FILES is what should be used for system settings files, not for user settings. 
The usual way to do this is in a launcher: the user runs the launcher, the config is copied if needed, the app is launched. With multiple users, there is no mechanism which would copy a template config to each possible user settings directory.

Reply:
I now used the system() function to call the script on start and it seems to work

And it seems to not overwrite a modified config

Reply:
> Yeah sorry for that. USER_SETTINGS_FILES is a declaration with no automatic actions. GLOBAL_WRITABLE_FILES is what should be used for system settings files, not for user settings.
> The usual way to do this is in a launcher: the user runs the launcher, the config is copied if needed, the app is launched. With multiple users, there is no mechanism which would copy a template config to each possible user settings directory.

no problem, I could've also just read the documentation